crates/iota-transaction-builder/src/builder/mod.rs

```diff
@@ -328,6 +328,56 @@ impl<C, L> TransactionBuilder<C, L> {
     }
 
     /// Transfer objects to a recipient address.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use std::str::FromStr;
+    /// # use iota_types::{Address, Digest, Transaction, ObjectId, ObjectReference};
+    /// # use iota_transaction_builder::{TransactionBuilder, res};
+    ///
+    /// # #[tokio::main(flavor = "current_thread")]
+    /// # async fn main() -> eyre::Result<()> {
+    ///
+    /// let client = iota_graphql_client::Client::new_devnet();
+    /// let sender =
+    ///     Address::from_str("0x611830d3641a68f94a690dcc25d1f4b0dac948325ac18f6dd32564371735f32c")?;
+    ///
+    /// let mut builder = TransactionBuilder::new(sender).with_client(client);
+    ///
+    /// # builder
+    /// #     .split_coins(
+    /// #         ObjectId::from_str(
+    /// #             "0x0b0270ee9d27da0db09651e5f7338dfa32c7ee6441ccefa1f6e305735bcfc7ab",
+    /// #         )?,
+    /// #         [1000u64],
+    /// #     )
+    /// #     .name(("coin"));
+    ///
+    /// builder.transfer_objects(
+    ///     Address::from_str("0x0000a4984bd495d4346fa208ddff4f5d5e5ad48c21dec631ddebc99809f16900")?,
+    ///     (
+    ///         // ObjectIds can be passed when a client is provided
+    ///         ObjectId::from_str(
+    ///             "0xd04077fe3b6fad13b3d4ed0d535b7ca92afcac8f0f2a0e0925fb9f4f0b30c699",
+    ///         )?,
+    ///         // ObjectReferences are always allowed, though they must be correct
+    ///         ObjectReference {
+    ///             object_id: ObjectId::from_str(
+    ///                 "0x8ef4259fa2a3499826fa4b8aebeb1d8e478cf5397d05361c96438940b43d28c9",
+    ///             )?,
+    ///             digest: Digest::from_str("4jJMQScR4z5kK3vchvDEFYTiCkZPEYdvttpi3iTj1gEW")?,
+    ///             version: 435090179,
+    ///         },
+    ///         // The result of a previous command can also be used
+    ///         res("coin"),
+    ///     ),
+    /// );
+    ///
+    /// let txn: Transaction = builder.finish().await?;
+    /// # Ok(())
+    /// # }
+    /// ```
```

Comment:
Put an equivalent of these two in the FFI crate?

Reply:
I dunno if it makes sense considering it's language agnostic, but I do think it would be nice to have some docs somehow